package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.76.0",
-  "distro": "ea016b36988a95ad53a905e9449f360c77fe5026",
+  "distro": "cc668e8e4391df1cd95743dc821c47a0889ddb50",
```

Comment:
The latest is cd6d83a02668b3ee1577d32ede0540c6e71b8119, did you mean that?

Reply:
Absolutely, thanks for double checking! Still not used to squash changing commits...